trench/command/deactivate_mfa_method.py

```diff
@@ -23,7 +23,7 @@ def execute(self, mfa_method_name: str, user_id: int) -> None:
             raise MFANotEnabledError()
 
         self._mfa_model.objects.filter(user_id=user_id, name=mfa_method_name).update(
-            is_active=False
+            is_active=False, is_primary=False
```

Comment:
Let's assume that there are 3 active MFA methods and the primary one is being deactivated. Don't we want to automatically set one of the two active methods that are left as primary here?

Reply:
This problem is solved in [line 20](https://github.com/merixstudio/django-trench/blob/1c5f150717f5c0308b3d1bd903abb70d56d0969e/trench/command/deactivate_mfa_method.py#L20). If we have more than one active method, it is not possible to deactivate the primary one. The problem was only when we have one method.

Reply:
Ok, cool 👍